presto-native-execution/presto_cpp/main/PeriodicTaskManager.cpp

```diff
@@ -145,6 +145,12 @@ void PeriodicTaskManager::start() {
           REPORT_ADD_STAT_VALUE(
               kCounterAllocatedMemoryBytes,
               (allocator->numAllocated() * 4096l));
+          if (auto* mmapAllocator =
+                  dynamic_cast<velox::memory::MmapAllocator*>(allocator)) {
+            REPORT_ADD_STAT_VALUE(
+                kCounterMappedMemoryRawAllocBytesSmall,
```

Comment:
```
s/kCounterMappedMemoryRawAllocBytesSmall/kCounterMallocAllocBytes/
```
Eventually we should get the stats from MemoryAllocator? Thanks!

Reply:
I'm using an already created but unused counter. Do you think we can get this in and refactor all-together when the unification of APIs happens in allocators? Thanks!